INSTALL

```diff
@@ -459,9 +458,13 @@ MFEM_USE_UMPIRE = YES/NO
    discovery, provision, and management of memory on machines with multiple
    memory devices like NUMA and GPUs.
 
+MFEM_USE_CAMP = YES/NO
```

Comment:
Is there some required CAMP code needed here for MFEM to support Umpire >= 3.0.0 ?

Reply:
I believe both RAJA and Umpire use CAMP in some capacity.

Reply:
Sure, but that's Umpire/Raja tpl requirements that should be pulled/compiled by their build system - if we don't have any code here?

Reply:
At least with Umpire we need the CAMP include flags when building MFEM with gmake